### ISSUE TITLE
Deleting an array of IDs gives a parameter error

### ISSUE THREAD

**Issue by bhaden94** [sanity-io/client#448]
**Describe the bug**

I am attempting to run a delete for an array of image document IDs. This seems to be possible by looking at the [code for delete operations](https://github.com/sanity-io/client/blob/2332016c7a98627990741c8f13aed18d2c0b1423/src/SanityClient.ts#L499C3-L502C35).

It looks like in the [Definition of getSelection](https://github.com/sanity-io/client/blob/main/src/util/getSelection.ts), it is not looking for the `id` field for the string array of IDs like the delete command expects to be passed in as defined in [MutationSelection type](https://github.com/sanity-io/client/blob/2332016c7a98627990741c8f13aed18d2c0b1423/src/types.ts#L428).

Here is the code I am trying to run:

```js
  const documentIds: string[] = ['id-1', 'id-2'] // IDs are verified to exist before running

  await client.delete({ id: documentIds }).catch((err) => {
    console.error(err)
  })
```
And here is the error that is output:
```sh
 ⨯ Error: Unknown selection - must be one of:

* Document ID (<docId>)
* Array of document IDs
* Object containing `query`
    at getSelection (webpack-internal:///(rsc)/./node_modules/@sanity/client/dist/_chunks/browserMiddleware-MafB5TmI.js:171:11)
    at _delete (webpack-internal:///(rsc)/./node_modules/@sanity/client/dist/_chunks/browserMiddleware-MafB5TmI.js:904:25)
    at _SanityClient.delete (webpack-internal:///(rsc)/./node_modules/@sanity/client/dist/_chunks/browserMiddleware-MafB5TmI.js:1786:116)
    at DELETE (webpack-internal:///(rsc)/./src/app/api/sanity/images/route.ts:47:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async D:\repos\nextjs-bookings\node_modules\next\dist\compiled\next-server\app-route.runtime.dev.js:6:62609
```

If I loop over the IDs array and run a delete on each individual one, it works fine. However, this means I would send an API request for every document being deleted, which is not ideal.

**To Reproduce**

1. Call the delete operation on the SanityClient by passing in and array of IDs in the format of the MutationSelection type. E.g. `{id:[...ids]}`
2. See an error output that says this is not a valid selection.

**Expected behavior**

For the delete operation to accept a list of IDs and delete them all.

**Which versions of Sanity are you using?**

Relevant versions from my `package.json`.
```
"@sanity/client": "6.8.6",
"@sanity/demo": "1.0.2",
"@sanity/vision": "3.20.0",
"sanity": "3.20.0",
"sanity-plugin-iframe-pane": "2.6.1",
"sanity-plugin-media": "^2.2.4",
```

**What operating system are you using?**
Windows 11.

**Which versions of Node.js / npm are you running?**

npm: 10.2.4
node: v20.9.0

**Comment by rexxars** [comment]
I believe the syntax here is:

```ts
const documentIds: string[] = ['id-1', 'id-2'] // IDs are verified to exist before running

await client.delete(documentIds).catch((err) => {
  console.error(err)
})
```

If you've got suggestions for a clearer way to phrase this error message, please let us know!

**Comment by bhaden94** [comment]
This is the error I get when passing in an array of strings directly:
```
No overload matches this call.
  Overload 1 of 10, '(id: string, options?: BaseMutationOptions | undefined): Promise<SanityDocument<Record<string, any>>>', gave the following error.
    Argument of type 'string[]' is not assignable to parameter of type 'string'.
  Overload 2 of 10, '(selection: MutationSelection, options?: BaseMutationOptions | undefined): Promise<SanityDocument<Record<string, any>>>', gave the following error.
    Argument of type 'string[]' is not assignable to parameter of type 'MutationSelection'.ts(2769)
```

**Comment by rexxars** [comment]
You're right, something is off here. Will investigate.

**Comment by rexxars** [comment]
Sorry about the very late follow up here - this has been fixed in [v6.13.1](https://github.com/sanity-io/client/releases/tag/v6.13.1)
